dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -60,16 +60,16 @@ class DALIGenericIterator(object):
     ----------
     pipelines : list of nvidia.dali.pipeline.Pipeline
                 List of pipelines to use
-    output_map : list of str
-                 List of strings (either "data" or "label") which maps
-                 the output of DALI pipeline to proper field in MXNet's
-                 DataBatch
+    output_map : list of (str, str)
+                 List of pairs (output_name, tag) which maps consecutive
+                 outputs of DALI pipelines to proper field in MXNet's
+                 DataBatch.
+                 tag is one of DALIGenericIterator.DATA_TAG
+                 and DALIGenericIterator.LABEL_TAG mapping given output
```

Comment:
Maybe it should be GPU_TAG and CPU_TAG, now if we want to have CPU data we need to use LABEL_TAG.

Reply:
It is not about being GPU or CPU here.
MXNet's data input structure makes a distinction between Data and Label. Model implementations expect labels to be set in the `label` NDarray list and Input data in the `data` ndarray.
https://mxnet.incubator.apache.org/api/python/io/io.html#mxnet-io-helper-classes-functions

Reply:
Understood.